server/controller/user-contoller.js

```diff
@@ -10,8 +10,9 @@ const getAllUser = async(req,res,next) =>{
     catch(err){
         console.log(err);
```

Comment:
The error object 'err' should not be directly logged as it may contain sensitive information. Consider logging a generic error message or sanitizing the error object before logging.

```suggestion
console.error("An error occurred");
```